examples/aks-terraform/cnab/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:latest
+
+ENV TERRAFORM_VERSION=0.11.8
+ENV TERRAFORM_SHA256SUM=f991039e3822f10d6e05eabf77c9f31f3831149b52ed030775b6ec5195380999
+
+RUN apk add --update git curl openssh bash-completion && \
+    curl https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip > terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+    unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /bin && \
+    rm -f terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+    apk update && \
+    apk add bash py-pip && \
+    apk add --virtual=build gcc libffi-dev musl-dev openssl-dev python-dev make && \
+    pip install --upgrade pip && \
+    pip install azure-cli
+
+COPY Dockerfile /cnab/Dockerfile
+COPY app /cnab/app
+
+RUN chmod 755 /cnab/app/run
```

Comment:
Tried to do a `duffle build` on this directory:

```
Step 5/9 : COPY Dockerfile /cnab/Dockerfile
 ---> abb6b13c59c9
Step 6/9 : COPY app /cnab/app
 ---> 6621ce9616c0
Step 7/9 : RUN chmod 755 /cnab/app/run
 ---> Running in 447135ad9430
chmod: /cnab/app/run: No such file or directory
Error: error building image: error building image cnab: error streaming messages for image builder cnab with builder docker: The command '/bin/sh -c chmod 755 /cnab/app/run' returned a non-zero code: 1
```

Reply:
oops, missed this and the init-backend when i added files, my bad!